.travis.yml

```diff
@@ -6,10 +6,13 @@ scala:
 jdk:
   - oraclejdk8
 env:
-  # Setting this variable twice will cause the 'script' section to run twice with the respective env var invoked
-  - BUILD_TYPE=sbt
-  - BUILD_TYPE=centaurJes
-  - BUILD_TYPE=centaurLocal
+  global:
+    - CENTAUR_BRANCH=79dc5f9f0f6ca6aedb5b13358d47b0c9c4156d51
```

Comment:
Yes-ish. To be explicit-- one can use a hash, a named-branch, or a tagged commit.

In general across our repos, currently our git tags point to the first hash that released a set of changes. There as fixes are added, the tag is not updated to the new hashes. Meanwhile, the branch name "xyz_hotfix" points to the latest and greatest hash with bug fixes for that release xyz.